### PR TITLE
New scripts for demo

### PR DESCRIPTION
**Pull request by StevenLWill**

### CODE REVIEW

**Comment by StevenLWill** [comment]
This code diff shows the implementation of a basic `Calculator` class in three different programming languages: C#, Python, and Swift. Each implementation includes two methods—`add` and `subtract`. Here's an analysis of potential improvements, best practices, and any issues:

### Overall Structure
1. **Maintain Consistent Naming Conventions**: 
   - Each implementation adheres to the respective language's naming conventions well. Ensure that the class and method names are aligned with standard practices in each language (e.g., PascalCase for C# and Swift, snake_case for Python).

2. **Separation of Concerns**:
   - The implementations are simple and keep functionality separated. If the calculator were to be extended, consider encapsulating functionalities in separate classes/modules to maintain clarity and manageability.

### C# Implementation
```csharp
using System;

public class Calculator
{
    public int Add(int a, int b)
    {
        return a + b;
    }

    public int Subtract(int a, int b)
    {
        return a - b;
    }
}
```

#### Feedback for C#
- **Error Handling**: Although not strictly necessary for a simple calculator, consider adding error handling for potential overflow in the `Add` and `Subtract` methods using `checked` context.
- **Unit Testing**: Ensure that unit tests are created for this class to validate its functionality, particularly for edge cases.
- **XML Documentation Comments**: Adding XML comments above the class and methods could enhance readability and provide quick references for users of your code.

### Python Implementation
```python
def add(a, b):
    return a + b

def subtract(a, b):
    return a - b
```

#### Feedback for Python
- **Function Annotations**: Consider using type hints for function parameters and return values for better clarity and to aid static analysis tools.
   ```python
   def add(a: int, b: int) -> int:
       return a + b
   ```
- **Error Handling**: Similar to the C# example, consider implementing error handling to address cases that might lead to runtime errors.
- **Encapsulation in a Class**: If this were to become a larger application, structuring the functions inside a `Calculator` class would maintain coherence with the C# and Swift implementations.

### Swift Implementation
```swift
import Foundation

class Calculator {
    func add(_ a: Int, _ b: Int) -> Int {
        return a + b
    }

    func subtract(_ a: Int, _ b: Int) -> Int {
        return a - b
    }
}
```

#### Feedback for Swift
- **Documentation**: Like the C# version, consider adding documentation comments for better understanding and usage.
- **Error Handling**: Implement checks for potential overflow conditions, though not common in typical applications unless working with very large integers.
- **Using Swift’s Standard Libraries**: Instead of manually handling potential edge cases, leveraging Swift's standard libraries or extensions could be beneficial if the application scales.

### General Suggestions
- **Consistency**: If these implementations are for the same project, ensure consistent design patterns across different languages. Decide whether to use classes or standalone functions and stick with that across all implementations.
- **Testing**: Regardless of the language, implementing unit tests is crucial. Each implementation should have corresponding tests to cover various scenarios and edge cases.
- **Performance Considerations**: Although this code is simple, be mindful of performance when scaling to larger, more complex calculations. 

In summary, the implementations are straightforward and functional, but they can be enhanced with considerations for error handling, type annotations, and documentation. Adhering to best practices for each language will result in a more robust and maintainable code.
